README.md

```diff
@@ -182,19 +148,22 @@ installation time by passing an options file to the DCOS CLI.
   "required": ["foo"]
 }
 ```
-_Sample `config.json`._
 
-`config.json` must be a valid [JSON Schema](http://json-schema.org/) file. Check out the
-[JSON Schema examples](http://json-schema.org/examples.html).
 
 #### `marathon.json.mustache`
+|Packaging Version|   |
+|-----------------|---|
+|2.0|required|
+|3.0|optional|
 
-This file describes how to run the package as a
-[Marathon](http://github.com/mesosphere/marathon) app.
+This file is a [mustache template](http://mustache.github.io/) that when rendered will create a
+[Marathon](http://github.com/mesosphere/marathon) app definition capable of running your service.
 
-User-supplied metadata (as described in `config.json`), the defaults from `config.json` and the
-resource information in `resource.json` will be injected to the template using
-[mustache template](http://mustache.github.io/) syntax.
+Variables in the mustache template will be evaluated from a union object created by merging three objects in the
+following order:
+1. Defaults specified in `config.json`
+2. User supplied options from either the DC/OS CLI or the DC/OS UI
+3. The contents of `resource.json`
```

Comment:
This list is not being rendered as an HTML `ol`. Maybe a line break is needed before item 1?